service/builder.go

```diff
@@ -22,31 +22,25 @@ package service
 
 import "github.com/pkg/errors"
 
+// WithModules is a helper to create a service without any options
+func WithModules(modules ...ModuleCreateFunc) *Builder {
```

Comment:
Well, builder pattern! This might be more suitable in builder package, but then we'll dive into circular dependency issues.